config/environment.js

```diff
@@ -232,6 +232,8 @@ module.exports = function(environment) {
         },
         featureFlagNames: {
             routes: {
+                'branded': 'ember_registries_detail_page', //TODO Replace with another flag
+                'branded.discover': 'ember_registries_detail_page', //TODO Replace with another flag
```

Comment:
Should I create a new feature flag here? Right now I'm just piggybacking off of the `ember_registries_detail_page` so I can see the route. When I added a new one `branded_registries`, the route didn't show. Should I be registering new feature flags somewhere?

Reply:
when using mirage, all these flags should be enabled by default -- oh, but when you update the config you have to kill your `ember s` and restart, it doesn't get pulled into rebuilds, unfortunately

Reply:
also these should be `registires.branded` and `registries.branded.discover` -- the engine is mounted at `registries`

unsure if that explains your problem with adding a new flag (my above comment might not be relevant).

it *should* be perfectly ok to add a new flag here, and it should be enabled in mirage -- when we want to use it with the backend we'll either enable the flag for real or remove these lines from the config